app/containers/LeftMenu/Documentation/Documentation.tsx

```diff
@@ -139,17 +139,25 @@ const Documentation: React.FC<DocumentationMenuSectionProps> = ({
         {Boolean(isModeratorModeSingleCommunity) && (
           <div className="dropdown-documentation db mr4">
             <Dropdown
-              trigger={
+              id="documentation_dropdown_id"
```

Comment:
check if this prop is needed and delete if not

Reply:
yes, this is a required prop